src/pubtools/_marketplacesvm/cloud_providers/base.py

```diff
@@ -38,6 +38,15 @@ class MarketplaceAuth(TypedDict):
 class CloudCredentials:
     """The base class for a cloud provider credentials."""
 
+    def __init__(self, **kwargs):
+        """Initialize the ClourCredentials by filtering out extra args."""
```

Comment:
Nice typo >< 
Fixing it

Reply:
It's [better now](https://github.com/release-engineering/pubtools-marketplacesvm/compare/0418bc9281d01f0d8a0a9ff20ecd4e2f591e3c62..aa7155cd7509887797937f449d0fa052cd600567)